### PR TITLE
made cards the same height for desktop and mobile 

### PR DESCRIPTION
**Pull request by AlexVCS**
change css styles for the card container and styles into tailwind styles.

The group descriptions are now scrollable.

Desktop:

localhost_3000_

Tablet (I still want to work on this and improve it):

localhost_3000_ (1)

Mobile:

localhost_3000_ (2)

### CODE REVIEW

File: components/Card/Card.module.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
-.card {
+/* .card {
```
Comment (jonathans199) [inline components/Card/Card.module.css:1]: @AlexVCS maybe we could remove this commented out code, since we are using Tailwind?

Comment (AlexVCS) [reply components/Card/Card.module.css:1]: good point @jonathans199! we can remove the whole file now

**Comment by zarela** [comment]
@AlexVCS is it possible to do something so people know is scrollable? or perhaps will be easier to have a button with a `...` that expands to the entire size of the card? 
Because right now people don't really know is scrollable, unless we give them a hint. Another reason I like the `...` button is because they know there is more, because right now on mobile for example looks like the sentences are incomplete.

If we decide to keep the scrollable which is fine too, just as long as they know it is scrollable is fine but perhaps we can do a smaller paddings for the top and bottom of the new bar that holds the socials.

Any feedback from anyone will be appreciated :)

**Comment by AlexVCS** [comment]
> @AlexVCS is it possible to do something so people know is scrollable? or perhaps will be easier to have a button with a `...` that expands to the entire size of the card? Because right now people don't really know is scrollable, unless we give them a hint. Another reason I like the `...` button is because they know there is more, because right now on mobile for example looks like the sentences are incomplete.
> 
> If we decide to keep the scrollable which is fine too, just as long as they know it is scrollable is fine but perhaps we can do a smaller paddings for the top and bottom of the new bar that holds the socials.
> 
> Any feedback from anyone will be appreciated :)

I am working on making it more clear for the user. I added custom scrollbar colors, this is a good addition for the desktop view. 

> @AlexVCS is it possible to do something so people know is scrollable? or perhaps will be easier to have a button with a `...` that expands to the entire size of the card? Because right now people don't really know is scrollable, unless we give them a hint. Another reason I like the `...` button is because they know there is more, because right now on mobile for example looks like the sentences are incomplete.
> 
> If we decide to keep the scrollable which is fine too, just as long as they know it is scrollable is fine but perhaps we can do a smaller paddings for the top and bottom of the new bar that holds the socials.
> 
> Any feedback from anyone will be appreciated :)

Thanks for your feedback Zarela. I pushed an update and am curious what you think. Still need to work on the social icons when there are more than four, but check out how the cards behave and the icon that shows up to try and help let the users know it is scrollable.

**Comment by zarela** [comment]
I will wait for other's feedback. I am still confused as how they scroll because only if a person hovers over will know you can do that, also the arrows, seems like is a button but I can never click on it.
Last thing, not sure what happened with the icons? They are all separated
Screenshot 2024-10-25 at 10 40 58 AM

Can they be like they were before?
Screenshot 2024-10-25 at 10 40 46 AM

**Comment by CrystalWernicke** [comment]
From my view, the social media buttons look like they are being cropped and either have flat bottoms or sides.

Computer view:
- 	Coders' Facebook Button
- 	Hackerspace's LinkedIn Button
- 	SWFLSec's Facebook Button
- 	Tech Night's LinkedIn Button
- 	VR & AR's Facebook Button

Mobile & Tablet view:
- 	Music Producers LinkedIn
- 	Python's Twitter
- 	Coders' Meetup, Facebook (bottom and right side), Twitter, LinkedIn, and GitHub
- 	Hackerspace's Facebook
- 	SWFLSec's Facebook
- 	Tech Night's Facebook
- 	VR & AR Meetup, Facebook (bottom and right side), Twitter, and LinkedIn

Other than that, I am satisfied with the cards themselves and lgtm.

**Comment by zarela** [comment]
closes #51

**Pull request merged**